libs/sdk-model/src/objRef/index.ts

```diff
@@ -45,6 +45,7 @@ export type ObjectType =
     | "tag"
     | "insight"
     | "variable"
+    | "prompt"
```

Comment:
I am not sure we want to expose/support this. Is this necessary?

Reply:
Probably not using now, can i delete it from tests or also from enum?